src/test/js/multibranch/testResults.js

```diff
@@ -27,9 +27,14 @@ module.exports = {
     },
 
     'Open acitivty page wait for first run to finish': (browser) => {
+        
+
         const blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName);
+        blueActivityPage.click(".branches");
+        blueActivityPage.click(".activity");
```

Comment:
Not sure if it's needed (might be covered internally in `click`), but I tend to add an explicit `waitForElementVisible` call before interacting (clicking etc).